Mathlib/RingTheory/DedekindDomain/Dvr.lean

```diff
@@ -43,7 +43,7 @@ dedekind domain, dedekind ring
 -/
 
 
-variable (R A K : Type*) [CommRing R] [CommRing A] [IsDomain A] [Field K]
+variable (A : Type*) [CommRing A] [IsDomain A]
```

Comment:
I'm unsure if `A` or `R` is a better name here. It's presumably fine to leave the current name as-is.

Reply:
The type that was used throughout was called `A` and it is a domain, rather than a simple commutative ring, so maintaining this notational distinction may be a good reason to avoid further changes.  Besides, Johan already merged the PR... :smile: